templates/3a_monitor_MIBI_run.ipynb

```diff
@@ -41,8 +41,8 @@
    "source": [
     "## Required variables\n",
     "\n",
-    " - `run_name` should contain the exact name of the MIBI run that will be monitored. \n",
-    " - `panel_path` should point to a panel csv specifying the targets on your panel. For details on panel formatting, see the [panel description](https://github.com/angelolab/toffy#panel-format)"
+    " - `run_name` should contain the exact name of the MIBI run that will be monitored\n",
+    " - `panel_path` should point to a panel csv specifying the targets on your panel (see the [panel format](https://github.com/angelolab/toffy#panel-format) for more information)"
```

Comment:
This is likely the first time they'll see this message, so I would add one more sentence. E.g. saying it needs to be downloaded from mibi tracker and copied into folder